docs/release-notes/0.12.0rc1.md

```diff
@@ -24,3 +24,4 @@
 - Adopt the Scientific Python [deprecation schedule](https://scientific-python.org/specs/spec-0000/) {user}`ilan-gold` ({pr}`1768`)
 - Allow {mod}`zarr` v3 writing of data {user}`ilan-gold` ({pr}`1892`)
 - {func}`anndata.register_anndata_namespace` functionality for adding custom functionality to an {class}`~anndata.AnnData` object {user}`srivarra` ({pr}`1870`)
+- Load AnnLoader lazily to prevent expensive unnecessary `torch` imports when its available on the system. {user}`Zethson` & {user}`flying-sheep` ({pr}`1959`)
```

Comment:
this (without the `-`) belongs into a file called `1959.performance.md` that you create using `hatch towncrier:create`

0.12.0rc1 is already released and its release notes are complete.

Reply:
Sorry, I don't know what I was thinking. Not even my first time using towncrier. Done.